src/OrchardCore/OrchardCore.Users.Abstractions/Services/IExternalLoginUserToRelateFinder.cs

```diff
@@ -0,0 +1,14 @@
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+
+namespace OrchardCore.Users.Abstractions;
+
+public interface IExternalLoginUserToRelateFinder
+{
+  bool CanManageThis(string extLoginKind);
```

Comment:
Maybe pass the ExternalLoginInfo instead of the string. Also rename the method to CanHandle() instead

Reply:
fixed in bd01473